web-client/app/styles/main.scss

```diff
@@ -1153,6 +1153,7 @@ ul.cards-list-container {
         margin: 10px;
         display: block;
         padding-top: 1em;
+        overflow: scroll;
```

Comment:
@mkhatib You don't need `word-wrap: initial;`. `overflow: scroll;` or `white-space: pre-wrap` is enough and it's not a Ubuntu only or highligh.js problem. 

To be make pre consistent everywhere you might need to check [this](https://github.com/ahmedelgabri/blank-global-typography/blob/master/typography.scss#L260-L267)

BTW, it has noting to do with FF on Ubuntu or Highlight.js cause I'm on Mac and FF has syntax highlighting but the text is not wrapping.